src/components/Theme/src/default-theme.js

```diff
@@ -45,11 +51,13 @@ export default function defaultTheme() {
 			fontFamily: 'inherit',
 			size: 0,
 			textColor: '@colors.text',
+			weight: 300,
```

Comment:
The default font weight for `regular` text is 400.

Reply:
The default for regular text according to Shimaa's design files is 300. I would ask her to clarify but she's out today. I'll ask tomorrow.

Reply:
Here's a code reference for it: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight and the design file here references 400 for paragraphs: https://www.figma.com/file/SHY7Z7PbcVssjIJVuJ3IwU/%F0%9F%8D%81-Maker?node-id=12421%3A6938

Reply:
i was referring to the actual typography project design files here: https://www.figma.com/file/RvCNKalYNiOB0iyw6jYJM7/Typography-system-%E2%80%94-Shimaa-%E2%80%94-Q421?node-id=603%3A41589

anyway, I talked to Shimaa and she's cool with bumping it to 400.